packages/webpack/src/runtime/runtime.js

```diff
@@ -261,6 +270,20 @@ const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
     // wrap webpack runtime for policy check and hardening
     const policyRequire = wrapRequireWithPolicy(__webpack_require__, resourceId)
 
+    /**
+     * Wrapped chunk loader
+     *
+     * @param {string | number} chunkId
+     */
+    policyRequire.e = (chunkId) => {
```

Comment:
does the type of `policyRequire` have an `e` prop as returned by `wrapRequireWithPolicy`?

Reply:
the type is  `@typedef {WebpackRequire & { [key: string]: any }} WrappedRequire` because it's a function that's specific and it has a bunch of stuff on it that's very internal to webpack.